src/libpriv/rpmostree-core.cxx

```diff
@@ -1333,7 +1333,7 @@ sort_packages (RpmOstreeContext *self, GPtrArray *packages, GCancellable *cancel
         if (!find_pkg_in_ostree (self, pkg, self->sepolicy, &in_ostree, &selinux_match, error))
           return FALSE;
 
-        if (is_locally_cached)
+        if (is_locally_cached && !self->is_container)
```

Comment:
This bit would go away once we start using the new ostree core support for bare-split-xattrs right?

Reply:
This is an interesting discussion. I think if we get write support for bare-split-xattrs via the C libostree APIs, we should evaluate whether to switch over fully to the core and e.g. drop `dnf_context_run` in favour of `rpmostree_context_assemble`. It may be that it's not worth it though, in which case we'd still need this.